src/pygenesis/find.py

```diff
@@ -0,0 +1,269 @@
+"""Implements find endpoint to retrieve results based on query"""
+
+import pandas as pd
+
+from pygenesis.http_helper import get_response_from_endpoint
+
+pd.set_option("max_colwidth", None)
+pd.set_option("expand_frame_repr", False)
+
+
+class Results:
+    """
+    A class representing the result object of variables, statistics, cubes and tables.
+
+    Attributes:
+        df (pd.DataFrame): The DataFrame that contains the data.
+        category (str): Category of the result. E.g. "tables", "cubes".
+
+    Methods:
+        get_code(): Gets code based on the index of the object.
+        get_metadata(): Gets metadata based on the index of the object.
+    """
+
+    def __init__(self, result: pd.DataFrame, category: str):
+        """
+        Class that contains the results of a find query.
+
+        Args:
+            DataFrame (pd.DataFrame): Result of a search query.
+            category (str): Category of the result. E.g. "tables", "cubes"
```

Comment:
warum ist category singular aber der Inhalt Plural?

Reply:
Beim parameter zum find enpunkt wird der Plural verwendet, bei der Url der Metadata Methode, der Singular. Das ist ein wenig unschön in der API umgesetzt, deswegen die Verwirrung. Haben das im Docstring nochmal klarer definiert.

Reply:
Danke!